service/src/main/java/bio/terra/landingzone/library/landingzones/definition/ResourceNameGenerator.java

```diff
@@ -26,7 +26,7 @@ public class ResourceNameGenerator {
   // errors as the cluster name is added to the node pool resource group name.
   // The node pool name resource group name can't be longer than 80 characters,
   // and follows this pattern: MC_resourceGroupName_resourceName_AzureRegion.
-  public static final int MAX_AKS_CLUSTER_NAME_LENGTH = 25;
+  public static final int MAX_AKS_CLUSTER_NAME_LENGTH = 18;
```

Comment:
Reducing for the increase in length of "southcentralus". We have a ticket to do more things around this-- WOR-1213.